src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -29,65 +25,42 @@
 import javax.servlet.http.HttpServletResponse;
 
 // Servlet that handles comments
-@WebServlet("/make-comment")
+@WebServlet("/comment")
 public class CommentServlet extends HttpServlet {
   /* This method takes comments, timestamps and the id of comments and creates a Task object.
    * It then converts the Task object to a json.
    */
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    Query query = new Query("Task").addSort("timestamp", SortDirection.DESCENDING);
-
-    DatastoreService savedComments = DatastoreServiceFactory.getDatastoreService();
-    PreparedQuery results = savedComments.prepare(query);
-
-    List<Task> tasks = new ArrayList<>();
-
-    for (Entity entity : results.asIterable()) {
-      long id = entity.getKey().getId();
-      long timestamp = (long) entity.getProperty("timestamp");
-      String comment = (String) entity.getProperty("comment");
-
-      Task task = new Task(id, timestamp, comment);
-      tasks.add(task);
+    List<Comment> comments;
+    if (getParameter(request, "store", "").isEmpty()) {
+      comments = CommentDatabase.getCommentByEmail(getParameter(request, "email", ""));
```

Comment:
@yjzhang111 I thought you changed getParameter so there is no need to pass "" as third parametere